hooks/attribute_checker.py

```diff
@@ -0,0 +1,37 @@
+import os
+import time
+import shutil
+
+def pre_export(output, conanfile, conanfile_path, reference, **kwargs):
+    # Check basic meta-data
+    for field in ["url", "license", "description"]:
+        field_value = getattr(conanfile, field, None)
+        if not field_value:
+            output.warn("Conanfile doesn't have '%s'. It is recommended to add it as attribute"
+                        % field)
+
+def pre_build(output, conanfile, **kwargs):
+    assert conanfile
+    if conanfile.in_local_cache:
+        output.info("reference={}".format(kwargs["reference"].full_str()))
+        output.info("package_id={}".format(kwargs["package_id"]))
+    else:
+        output.info("conanfile_path={}".format(kwargs["conanfile_path"]))
+
+        destPath = kwargs["conanfile_path"]
+        for x in range(2):
+            destPath = destPath[:destPath.rfind('/')]
```

Comment:
przeczuwam tu problem z prawidłową ścieżką, ale może będzie ok

Reply:
ogólnie to działa ;)